julia/oxygen/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  github: OxygenFramework/Oxygen.jl
+  version: 1.5.1
```

Comment:
Is the `version` property required?  If possible, I'd like to keep things as generic as possible so that it always pulls the latest version. Granted, I don't know if this is best practice in the benchmarking world but It would certainly make my life easier since I wouldn't have to make as many followup PR's for new versions

Reply:
This is just a descriptive version (to have this info displayed on results) and not the version installed